### ISSUE TITLE
Sessions not rewarded:

### ISSUE THREAD

**Issue by Sendros** [mysteriumnetwork/node#5444]
Sessions not rewarded:

![image](https://user-images.githubusercontent.com/105081591/190872981-1e572858-7726-4cf9-b06b-072578fc4956.png)
![image](https://user-images.githubusercontent.com/105081591/190873015-c62c9c40-faa8-4bbd-b706-9353f848b3b6.png)

There was no settlement done between these sessions. so where did these rewards go?

**Comment by Sendros** [comment]
![image](https://user-images.githubusercontent.com/105081591/190928128-9d9498ef-ce89-47c6-9dfc-9406fc10dbed.png)
Also not rewarded.....

**Comment by etherunit** [comment]
Hello @Sendros.
It's still valid? No changes on the auto-settlement bar?

**Comment by Sendros** [comment]
@etherunit  Still nothing. Have to say that this new UI Makes it hard to follow.

**Comment by n10ty** [comment]
@Sendros could you please provide your Mysterium ID?

**Comment by Sendros** [comment]
> 

0x55cdaabf560c46b1cada8d9372f5ddd2ed6e3271
